homework/index.js

```diff
@@ -30,15 +32,73 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function renderContributors(contributors, contributorContainer) {
+    contributors.forEach(contributor => {
+      const contributorDetail = createAndAppend('div', contributorContainer, {
+        class: 'contributor-detail',
+      });
+      createAndAppend('img', contributorDetail, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'avatar',
+      });
+      createAndAppend('a', contributorDetail, {
+        text: contributor.login,
+        href: contributor.html_url,
+        target: '_blank',
+      });
+      createAndAppend('p', contributorDetail, { text: contributor.contributions });
+    });
+  }
+
+  function dropDown(root, repos) {
+    const header = createAndAppend('header', root, { class: 'header' });
+    createAndAppend('h3', header, { text: 'HYF Repositories' });
+
+    const select = createAndAppend('select', header);
+    repos.forEach((repo, index) => {
+      createAndAppend('option', select, {
+        text: repo.name,
+        value: index,
+      });
     });
+
+    const container = createAndAppend('div', root, { id: 'container' });
+    const repoContainer = createAndAppend('div', container, { id: 'repo-container' });
+    const contributorContainer = createAndAppend('div', container, { id: 'contributor-container' });
```

Comment:
There is one thing missing from this week's homework which was working in Week1 which is fetching the first repositories data by default on page load. You did it using this earlier:

```
fetchAndRender(repos[0], repoContainer, contributorContainer, root);
```

but it isn't there in this week's homework. Could you please look into it?